Makefile

```
@@ -82,6 +89,23 @@ require:
 	@which tmux >/dev/null 2>&1 || (echo "ERROR: tmux is required."; exit 1)
 	@which finger >/dev/null 2>&1 || (echo "ERROR: finger is required."; exit 1)
 
+guard-%:
+	@if [ -z '${${*}}' ]; then echo 'ERROR: .env variable $* not set' && exit 1; fi
```

For some reason, with single quotes here it doesn't do the check properly, and I get:

```
$ make validate.env
==> Validating your .env file for make up
/bin/sh: line 0: [: too many arguments
.env valid for make up
```

But with double-quotes, it works for me:

```suggestion
	@if [ -z "${${*}}" ]; then echo 'ERROR: .env variable $* not set' && exit 1; fi
```